### PR TITLE
Fix circular ref between mesh and nodes and elements

### PR DESCRIPTION
**Pull request by cbellot000**

### CODE REVIEW

File: ansys/dpf/core/collection.py [hunk 1]
```diff
@@ -41,11 +41,6 @@ class Collection:
         default is ``None``, in which case an attempt is made to use the global
         server.
 
-    Examples
```
Comment (PProfizi) [inline ansys/dpf/core/collection.py:44]: @cbellot000 Why was this one problematic?

File: ansys/dpf/core/meshed_region.py [hunk 2]
```diff
@@ -133,9 +133,7 @@ def elements(self):
         DPF Elements object with 8 elements
 
         """
-        if self._elements is None:
-            self._elements = Elements(self)
-        return self._elements
```
Comment (PProfizi) [inline ansys/dpf/core/meshed_region.py:138]: @cbellot000 That was a quick fix for a problem with vtk grids.

File: ansys/dpf/core/collection.py (hunk 1 above)
Comment (cbellot000) [reply ansys/dpf/core/collection.py:44]: @PProfizi it's not a supported behavior we should use the implementation instead (MeshesContainer here)

File: ansys/dpf/core/meshed_region.py (hunk 2 above)
Comment (cbellot000) [reply ansys/dpf/core/meshed_region.py:138]: @PProfizi what do you mean? I didn't take the fix from anywhere, I'm fixing a circular ref here

Comment (PProfizi) [reply ansys/dpf/core/meshed_region.py:138]: @cbellot000 ah yes sorry I confused this with another part of code.

**Review by PProfizi — APPROVED** [review]

**Review by anslpa — APPROVED** [review]

**Pull request merged**